mailet/api/pom.xml

```diff
@@ -55,11 +55,16 @@
             <version>1.7.5</version>
```

Comment:
as I suggested, you can parse it only when asking for, so you'll benefit of a cache when nothing changed